design-patterns/easy-circulars/ui/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "easy-circulars",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@radix-ui/react-scroll-area": "^1.2.3",
+    "@radix-ui/react-select": "^2.1.6",
+    "@radix-ui/react-slot": "^1.1.2",
+    "@radix-ui/react-tabs": "^1.1.3",
+    "@react-pdf-viewer/core": "^3.12.0",
+    "axios": "^1.7.9",
+    "class-variance-authority": "^0.7.1",
+    "clsx": "^2.1.1",
+    "lucide-react": "^0.475.0",
```

Comment:
Let's follow up on use of lucid tomorrow. Probably, might require switching to MUI for consistency. cc @Ervin0307